.drone.yml

```diff
@@ -0,0 +1,83 @@
+---
+kind: pipeline
+type: kubernetes
+name: pr
+
+steps:
+  - name: fetch tags
+    image: docker:git
+    commands:
+      - git fetch --tags
+  - name: wait for docker
+    image: docker
+    commands:
+      - timeout 15s /bin/sh -c 'while [ ! -S /var/run/docker.sock ]; do sleep 1; done'
+      - docker version
+    volumes:
+      - name: dockersock
+        path: /var/run
```

Comment:
Do you need the whole run directory or will just the docker socket file suffice?

Reply:
It needs to be the directory, because the same volume needs to be mounted to `docker:dind` before the socket exists.